RecipeApp/RecipeApp/Program.cs

```diff
@@ -1,5 +1,7 @@
 ﻿// Import classes from system namespace
+
 using System;
+using System.Collections.Generic;
```

Comment:
FYI: For pure expediency's sake I will not be reviewing anything other than HW2 content in this review.